pkg/cmd/actions/actions.go

```diff
@@ -34,28 +34,45 @@ func NewCmdActions(f *cmdutil.Factory) *cobra.Command {
 	return cmd
 }
 
-func actionsRun(opts ActionsOptions) {
-	cs := opts.IO.ColorScheme()
-	fmt.Fprint(opts.IO.Out, heredoc.Docf(`
-			Welcome to GitHub Actions on the command line.
+func actionsExplainer(cs *iostreams.ColorScheme, color bool) string {
```

Comment:
Not sure how you feel about this, but we could just pass in `cs` and check if it is `nil` instead of passing in a second `bool` argument for determining if we should use color